test/src/process.js

```diff
@@ -16,9 +16,6 @@ test('process.exit()', (t) => {
 })
 
 test('process.cwd', async (t) => {
```

Comment:
Can  we add a test to ensure `cwd` exists?

Reply:
there are plenty tests for this already in `test/src/process.js`